theme.inc

```diff
@@ -95,3 +109,209 @@ function _gent_base_get_timeline_display(Paragraph $entity) {
   $isProgram = $entity->get('field_timeline_as_program')->value;
   return ($isProgram === '1') ? 'alt' : 'normal';
 }
+
+/**
+ * Count the number of input fields in a render array.
+ *
+ * @param array $array
+ *   Render array to loop over.
+ *
+ * @return int
+ *   The number of input fields counted.
+ */
+function _gent_base_count_input_fields(array $array) {
+  $input_count = 0;
+  $fields_types = [
+    'checkbox',
+    'checkboxes',
+    'date',
+    'email',
+    'file',
+    'number',
+    'password',
+    'radio',
+    'radios',
+    'select',
+    'textarea',
+    'textfield',
+    'url',
+    'webform_address',
+  ];
+
+  foreach ($array as &$field) {
+    if (!is_array($field) || !isset($field['#type'])) {
+      continue;
+    }
+
+    if (!in_array($field['#type'], $fields_types)) {
+      continue;
+    }
+
+    $input_count++;
+  }
+
+  return $input_count;
+}
+
+/**
+ * Adds the step title to the step for webforms with multiple pages.
+ *
+ * @param array $form
+ *   Render array for the form.
+ */
+function _gent_base_add_webform_step_title(array &$form) {
+  foreach (Element::children($form['elements']) as $key) {
+    $step = &$form['elements'][$key];
+
+    if ($step['#type'] !== 'container') {
+      continue;
+    }
+
+    // Generate the markup.
+    $html = '<h2>' . $step['#title'] . '</h2>';
```

Comment:
Create a render array with [html_tag elements](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21Element%21HtmlTag.php/class/HtmlTag/8.5.x).

Or better: use a template.

```php
$build = [];
$build['heading'] = [
  '#type' => 'html_tag',
  '#tag' => 'h2',
  '#value' => $step['#title'],
];

if (_gent_base_count_input_fields($step) > 0) {
  $build['disclaimer'] = [
    '#type' => 'html_tag',
    '#tag' => 'p',
    '#value' =>  t('Fill in all fields unless otherwise stated'),
    '#attributes' => ['class' => ['form-disclaimer']],
  ];
}

...
```